core/error.rs

```diff
@@ -104,6 +104,7 @@ pub struct JsError {
   pub frames: Vec<JsStackFrame>,
   pub source_line: Option<String>,
   pub source_line_frame_index: Option<usize>,
+  pub aggregated: Option<Vec<Box<JsError>>>,
```

Comment:
Redundant `Option<Vec<Box<T>>>`? It can just be a `Vec<T>`.

Reply:
It's not, this option allows distinction between AggregateError and regular Error.

Reply:
Oh, I meant the redundant use of the `Box<T>`. compile-time size shouldn't be an issue since Vec is heap-allocated.

Reply:
Good point. I'll remove these boxes